### PR TITLE
MEPTS-527: Update the DSD N2 Indicator

### PR DESCRIPTION
**Pull request by JoaoCumbula**

### CODE REVIEW

**Comment by amosh101** [comment]
Code looks good to me.

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/EriDSDCohortQueries.java [hunk 1]
```diff
@@ -767,15 +767,34 @@ public CohortDefinition getPatientsWithNextConsultationScheduled175To190Days() {
...
+            "onOrAfter=${endDate-18m},onOrBefore=${endDate},location=${location}"));
+
+    cd.addSearch(
+        "tbPatient",
+        EptsReportUtils.map(
+            commonCohortQueries.getPatientsOnTbTreatment(),
+            "startDate=${startDate},endDate=${endDate},location=${location}"));
+
+    cd.setCompositionString(
+        "TxCurr AND (scheduledN2 OR rapidFlow) NOT (completed OR pregnant OR breastfeeding OR tbPatient)");
```
Comment (psmatsinhe) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/EriDSDCohortQueries.java:797]: @JoaoCumbula please change this block of pregnant, breastfeeding and TB to 
```suggestion
        "TxCurr AND (scheduledN2 OR rapidFlow) NOT (completed OR PregnantAndBreastfeedingAndOnTBTreatment)");
```

**Comment by psmatsinhe** [comment]
> _No description provided._

@JoaoCumbula Before you commit make sure you run `mvn clean install`

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/EriDSDCohortQueries.java (hunk 1 above)
Comment (JoaoCumbula) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/EriDSDCohortQueries.java]: @cfaife Already did this change and fix the compile issues, all i did is fix the mapping on DataSet class. Pull request already sent.

**Pull request merged**